design/images/context.puml

```diff
@@ -3,13 +3,13 @@
 
 Person(user_data_input, "Data input")
 Person(user_data_request, "Data discover and request")
-Person(user_admin, "Admin")
+Person(user_admin, "Admin", "Data owners, data managers, and other ")
```

Comment:
If data "owners" and "managers" are added here, should they be described in the text as well?

Reply:
Good idea, I've synch'd the text